Added REST calls for the ExAC Browser

JREastonMarks opened the pull request:
Hi,
I added some REST calls to the ExAC browser to increase its already impressive set of features.

JREastonMarks committed:
Updated README

Updated the realm file to remove the TODO

JREastonMarks committed:
Bug Fix and Gene REST Services

Fixed loading bug and added Gene REST services

JREastonMarks committed:
Added REST Services

Added REST Services for Variants, Genes, Regions, and DBSNP

JREastonMarks committed:
Fixes

Added additional paths to ensure compatibility with awesome search.

JREastonMarks committed:
Merge branch 'rest-bug-dev'

JREastonMarks committed:
Updates

- Added no service feature to search.
- Updated calls to return more information for basic rest call.

JREastonMarks committed:
Merge pull request #1 from hms-dbmi/rest-bug-dev

Updates

JREastonMarks committed:
Updates to UI

Updated UI to allow for faster initial display and an overall better
experience.

JREastonMarks committed:
Merge branch 'rest-bug-dev'

JREastonMarks committed:
REST Doc

Added a list of available REST calls

JREastonMarks committed:
Fix Doc

Fixed the formatting of the REST API documentation

JREastonMarks committed:
Merge branch 'rest-bug-dev'

JREastonMarks committed:
Update Requirements.txt

Update the pymongo library version to 2.9 which does not have the batch error.

JREastonMarks committed:
Merge remote-tracking branch 'konradjk/master'

# Conflicts:
#	templates/gene.html
#	templates/layout.html

JREastonMarks committed:
Bulk processing

Added the ability to bulk process variants

JREastonMarks committed:
Merge remote-tracking branch 'origin/master'

JREastonMarks committed:
Fix Conflicts

Fix conflicts with ExAC version of gene and layout pages

JREastonMarks committed:
Updated README

Added the post call information to the README file

ivan-krukov commented:
The `hms` server seems to be turning down API requests today (July 5th). July 4th was quite fine. Any thoughts? http://exac.hms.harvard.edu/rest/awesome?query=CHD8

JREastonMarks commented:
Hi Ivan,
We had some hiccups with our server. It is back up and running with newest version.

~Jeremy

anthony-cros commented:
This is great!

shesketh commented:
Hi, I realise this is an old conversation, but the hms server appears to be down again, and has been for the last few days (I first noticed this on Friday July 14th 2017). Would appreciate an indication of when this will be back up and running! Thanks for the great add-on to ExAC :)

shesketh commented:
Hi, this server appears to be down at the moment, would appreciate any info on when it will be up and running again! Thank you for a very useful extension to ExAC :)

Commit:
Update script for newer version of python

Change extension naming to comply with modern standards.

Also add UNIX environment variables, with default values, to make it ready for Dockerization

Commit:
Update Flask components to latest version

Commit:
Restoring version specific requirements

namdets committed:
Merge pull request #4 from gaborkorodi-hms/patch-3

Update Flask components to latest version

namdets committed:
Merge pull request #3 from gaborkorodi-hms/patch-2

Update script for newer version of python
